packages/eslint-config-base/rules/exports.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  rules: {
+    // Do not prefer default exports
+    'import/prefer-default-export': 'off',
+
+    // Disallow default exports
+    'import/no-default-export': 'error',
+  },
+  overrides: [
+    {
+      files: [
+        // Storybook stories
+        '**/*.stories.{jsx,tsx}',
+
+        // Next.js pages and API routes
+        'src/pages/**/*.{jsx,tsx}',
+        '/pages/**/*.{jsx,tsx}',
```

Comment:
Got an error on this, it looks like `/pages` needs to be `pages` for me.

Reply:
Looks like I forgot a leading `/` on line 16 – I will add that.

Line 17 is there to support pages being at the root, since Next.js supports both `/src/pages/**` and `/pages/**`

Reply:
Update: Looks like ESLint does not like `/` in override paths – it automatically resolves paths based on the directory that holds the ESLint config. Removing both leading `/` characters should do the trick.

Reply:
Fixed in a96504a.